dist/routable-component.js

```diff
@@ -172,9 +172,9 @@ var routableComponent =
 	     */
 	
 	  }, {
-	    key: 'register_component',
-	    value: function register_component(name, component) {
-	      if (this.components[name]) this.unregister_component(name);
+	    key: 'registerComponent',
+	    value: function registerComponent(name, component) {
+	      if (this.components[name]) this.unregisterComponent(name);
```

Comment:
__[eslint]__
* Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
* Error - Expected indentation of 2 space characters but found 0. (indent)

<!-- Comment by [SideCI](https://sideci.com) -->